sass/components/_channel-switcher.scss

```diff
@@ -56,6 +56,120 @@
         }
     }
 
+    .channel-switcher__content {
+        .multi-select__container {
+            padding: 0 2.4rem;
+        }
+
+        .multi-select__help {
+            padding: 8px 2.4rem 0;
+        }
+
+        .filtered-user-list {
+            display: block;
+            height: auto;
+        }
+
+        .react-select__multi-value__label {
+            display: flex;
+            align-items: center;
+            font-size: 14px;
+            font-weight: 600;
+            padding: 4px;
+        }
+
+        .multi-select__wrapper {
+            padding: 12px 0;
+            height: auto;
+            overflow: auto;
+            position: absolute;
+            width: calc(100% - 48px);
+            margin: -28px 0 0 24px;
+            background: v(center-channel-bg);
+            box-shadow: $elevation-4;
+            border-radius: 4px;
+            border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+            max-height: 272px;
+            z-index: 5;
+        }
+
+        .btn-primary {
+            font-size: 14px;
+            font-weight: 600;
+            padding: 0 36px;
+            height: 40px;
+            border-radius: 4px;
+        }
+
+        .no-channel-message {
+            font-weight: 600;
+            padding: 1rem;
+            font-size: 1.4rem;
+
+            .primary-message {
+                font-size: inherit;
+                margin: 0;
+            }
+        }
+
+        .react-select__value-container {
+            padding: 0 16px;
+            font-size: 16px;
+
+            &.react-select__value-container--has-value {
+                padding: 0 8px;
+            }
+        }
+
+        .more-modal__list {
+            height: auto;
+
+            > div {
```

Comment:
let's keep this style semantic as it's a problem within webapp we have constantly crashing styles because of direct element styling.